mountainduck/connect/index.md

```diff
@@ -0,0 +1,100 @@
+Connect mode
+===
+
+```{toctree}
+:hidden:
+:titlesonly:
+sync
+```
+While connecting to a server, you can choose between *Online* mode and *[Smart Synchronization](sync.md)* mode.
+
+## Online Mode
+
+The *online* mode is the most basic connect mode. Files you open get stored in the temporary system cache and deleted at the latest when the bookmark is disconnected.
+
+### Transfer progress
+
+While transferring files and folders, a transfer window will popup showing the transfer progress. In case of small files, the transfer might be finished to quick to actually realise the popup.
+
+## Status of Files
+
+Files and folders on a mounted volume have a status icon overlay in File Explorer (Windows) and Finder (macOS).
+
+```{note}
+Please make sure to enable the Mountain Duck [Integration](../installation/index.md) in *System Preferences → Extensions → Finder* on macOS.
+
+For macOS Ventura and later, the setting can be found in *System Settings → Privacy & Security → Extensions → Added Extensions*.
+```
+
+### ![](../_images/overlay_infinite.png) Online Only
+The file can only be opened when a connection to the server or cloud storage can be made. The file does not take any space on your computer. The file is downloaded on demand when you open it.
+
+### ![](../_images/overlay_error.png) Sync Error
+There was an error updating the file after changes. You are missing permission to write to the file or another problem occured. Please contact your web hosting service provider for assistance. To resolve the error, move the file to your local disk, and reload the directory or use the _Retry_ option within the context menu. Refer to [Sync Conflicts](sync.md#sync-conflicts) for possible error scenarios.
+
+### ![](../_images/overlay_ignored.png) Ignored
+The file is only saved in a local [temporary](../issues/index.md#temporary-files) location and never synced to the server or cloud storage.
+
+## Sync paused
+
+You can manually pause uploads and indexing by selecting *Pause Sync* in the submenu for the sync status. Syncing is also paused automatically when your network connection to the server is interrupted but resumed automatically when a connection is restored.
+
+```{warning}
+As long as the sync status is paused, uncached files marked as [Online Only](#online-only) can't be opened. An error message will be shown by the application attempting to open the file and a *Access Denied* notification will show up.
+```
+
+## Sync Errors
+
+Files that failed to sync get a sync error badge. You can try to repeat the failed transfer by selecting *Mountain Duck → Retry* in the context menu.
+
+### Resolve Errors
+
+If a sync error cannot be solved using *Mountain Duck → Retry* because the server does not allow the operation (i.e. due to a permission issue), you can resolve the error state on the file or folder by
```

Comment:
either complete the sentence with "[...] by using one of the options below." or adjust the bullet points so that they form a dramatically correct sentence with the part of the sentence.